src/forms/FormFieldNumberTable/FormFieldNumberTable.styled.tsx

```diff
@@ -52,23 +25,34 @@ export const TrHead = styled.tr`
   th:first-child {
     padding-left: 24px;
   }
+
+  th:last-child {
+    padding-right: 24px;
+  }
 `;
 
 export const Td = styled.td`
-  padding: 16px 0;
+  padding: 16px 16px;
```

Comment:
First and last tds should also have the 24 of padding left / right respectively, so they can align with their titles.